### DISCUSSION TITLE
Visibility on any node of a blacklisted user when followed

### DISCUSSION THREAD

**Discussion by 0xNordian** [deso-protocol/dips#195] [Medium CIPs]
**name:** Visibility on any node of a blacklisted user when followed
**about:** Users following blacklisted users should see their content in any node.
**labels:** Node Blacklist, blacklisted Users, Engagement
**assignees:** N/A

### One-Line Description
Allow users that are following blacklisted users to see their content on any node.

### Explanation and Motivation
_Example:_ 
If I'm following Designista and one day Brootle decide to blacklist their account on Gemstori, the following should happen:
1) Their account shouldn't be reachable from the Gemstori node for users that are not yet following Designista.
2) My user should see Designista content on the Gemstori node if 'following' is TRUE and 'blocking' is FALSE. 

The motivation is that it doesn't make sense to expect users to jump around nodes to see the content of the people they are following if they have been blacklisted on the node they usually use.

### Detailed proposal:
**User:** Nordian
**Node:** Gemstori
**Target Account:** Designista

- Nordian regularly use Gemstori.
- Nordian follows Designista, who is not blacklisted on Gemstori; therefore, see their content on Gemstori.
- Gemstori blacklist Designista.
- Nordian, who is still following Desginista and haven't blocked them, can still see the content of Designista on Gemstori.
- After blacklist has happened, people that are not following Designista cannot find their profile through Gemstori.
- If the user doesn't want to continue seeing the content of Designista on Gemstori simply can proceed to unfollow/block that user.

### Some benefits:
- The user has the control to keep seeing the content they want on any Node.
- The user doesn't have a bad experience on Deso if some of their following users get blacklisted, not being forced to jump around nodes to engage with them.
- Node Admin can still blacklist users and not make reachable their content from a given point in time.
- Fosters better decentralization of the ecosystem.

### Trade-off:

- If a user has been blacklisted for a legal reason and the node is obliged to not make reachable any type of data after authority request, we will need to think of a 'stronger' blacklist solution for those cases.

**Comment by brootle** [comment]
That's great idea, but really there is no way to get protected, apps can decide themselves if they are going to respect this rule and show you blocked by their node users if you follow them. Anyway, as far as I know no one is blocked at best DeSo App Gemstori, you can easily see here https://gemstori.com/@Designsta

**Reply by therealmatreshka** [reply]
I would add to that 
If you try to unfollow a blacklisted user, you should get a warning you will not be able to find them and follow again if you change your mind. We all accidently unfollow people regularly somehow so would hate this to happen.

**Reply by 0xNordian** [reply]
Well, you will be able to follow them again if you search for them in another node where they are not blacklisted. But I agree that a warning to explain the consequences will be helpful.
